### PR TITLE
Added screens and basic redux

### PR DESCRIPTION
**Pull request by n-kumari**
##### Description

Added basics of other screens and redux. Since we don't have data in our project as of now, the redux is just some installations and folders.

### CODE REVIEW

**Comment by n-kumari** [comment]
@FuadBalashov @psteele2 Please review and merge.

File: src/store.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { Provider } from 'react-redux';
+import { createStore } from "redux";
+
+
+export default store;
```
Comment (steveccable) [inline src/store.js]: Is `store` actually defined here?

File: src/Components/SplashScreen/SplashScreen.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import { Text } from 'react-native';
 
-class SplashScreen extends Component {
+class SplashScreen extends React.Component {
```
Comment (steveccable) [inline src/Components/SplashScreen/SplashScreen.js]: Probably ought to take `Component` out of the import if you are using the `React.` part.

File: src/index.js [hunk 3]
```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import MapScreen from './Components/MapScreen/MapScreen';
+import SplashScreen from './Components/SplashScreen/SplashScreen';
```
Comment (steveccable) [inline src/index.js]: Doesn't appear to be getting used here.

File: src/store.js (hunk 1 above)
Comment (steveccable) [inline src/store.js]: Also we don't appear to be connecting the store to anything, or creating it anywhere.  Normally that is the sort of thing which happens in the entry point (which looks like index.js, i guess?)

File: src/index.js (hunk 3 above)
Comment (n-kumari) [reply src/index.js]: Ha! Yes. I was still finding a way to add splash screens other than adding them separately to android and ios. And I forgot to remove this import.

File: src/store.js (hunk 1 above)
Comment (n-kumari) [reply src/store.js]: Ah, my bad. I removed the code I was playing with around and forgot to remove few things. This is one of those.

Comment (n-kumari) [reply src/store.js]: Yes, we can have it in entry point but some redux tutorials had a separate folder for store too. I couldn't figure out which is a better way so I just kept it this way.

File: src/Components/SplashScreen/SplashScreen.js (hunk 2 above)
Comment (n-kumari) [reply src/Components/SplashScreen/SplashScreen.js]: Fair point 👍

File: src/Components/FeedbackScreen/FeedbackScreen.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    render() {
+        return (
+           <Text>
+            Please provide your valuable feedback
+           </Text>
+        )
+    }
+}
+
+export default FeedbackScreen;
```
Comment (psteele2) [inline src/Components/FeedbackScreen/FeedbackScreen.js]: (nit) Shouldn't this have an empty line at the end?

**Comment by n-kumari** [comment]
@psteele2 @steveccable  Addressed comments and resolved conflicts.

**Review by steveccable — APPROVED** [review]

**Comment by psteele2** [comment]
@n-kumari one more conflict with imports. Fix and I'll merge 🙃

**Comment by n-kumari** [comment]
@psteele2 Done

**Pull request merged**